docs/topics/native/native-definition-file.md

```diff
@@ -36,17 +36,12 @@ Let's create a definition file and generate bindings for a C library:
    ```
 
    * `headers` is the list of header files to generate Kotlin stubs. You can add multiple files to this entry,
-     separating each with a `\` on a new line. In this case, it's only `png.h`. The referenced files need to be available
-     on the system path (in this case, it's `/usr/include/png`).
+     separating each with a space. In this case, it's only `png.h`. The referenced files need to be available
+     on the specified path (in this case, it's `/usr/include/png`).
    * `headerFilter` shows what exactly is included. In C, all the headers are also included when one file references
      another one with the `#include` directive. Sometimes it's not necessary, and you can add this parameter
      [using glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)) to fine-tune things.
 
-     `headerFilter` is an optional argument and is mostly used when the library is installed as a system library. You don't
-     want to fetch external dependencies (such as system `stdint.h` header) into the interop library. It may be important to
```

Comment:
The part about fetching external dependencies into the interop library is still necessary, as well as about fixing conflicts.
[Here](https://github.com/JetBrains/kotlin-web-site/pull/4405#discussion_r1775094432) I meant removing only one sentence, not the whole paragraph.

Reply:
Ah, got it! included the paragraph again with some minor changes